backend/spell.py

```diff
@@ -6,9 +6,42 @@
 from backend.location import find_adjacent_hexes
 from backend.operation import Operation
 from graphics.screen_input import choose_from_list
+import graphics.screen_input as screen_input
```

Comment:
it's nicer to use one style per file, so either adding the specific things needed to the previous line or getting rid of the previous line and converting `choose_from_list`->`screen_input.choose_from_list`